configmaster/views.py

```diff
@@ -71,6 +72,15 @@ def post(self, request, *args, **kwargs):
 
 class DeviceGetVersionAPIView(View):
     def get(self, request, *args, **kwargs):
+        if not settings.CONFIGMASTER_STATUS_API_KEY:
```

Comment:
How about making `CONFIGMASTER_STATUS_API_KEY` optional so that the production setup does not have to get changed?

Reply:
Should be a trivial change on your side from what I know.

Reply:
This would also make it easier for users who don't have to deal with audits.